indico/web/client/styles/base/_layout.scss

```diff
@@ -135,3 +133,8 @@ body > * {
         width: 0;
     }
 }
+
+
+@mixin clearfix() {
+    overflow: hidden;
+}
```

Comment:
missing trailing linebreak

Reply:
Sorry, my editor was misconfigured at the time 😞